src/components/molecules/TheHeader.vue

```diff
@@ -117,6 +128,9 @@ export default {
       useI18n().locale.value == "fr" ? "en" : "fr"
     );
     return { changeLanguageTo };
+
+    const lang = computed(() => (useI18n().locale.value == "fr" ? "en" : "fr"));
+    return { lang };
```

Comment:
Doesn't seem like you used this.

Reply:
good catch. I will remove it as it isn't used.